app/utils/misc.ts

```diff
@@ -1,4 +1,6 @@
+import { ClassNameValue } from 'node_modules/tailwind-merge/dist/lib/tw-join.js'
```

Comment:
I don’t think relying on internals of a lib is a good idea

Reply:
Add a local ClassNameValue type instead of relying on the internals